docs/json.md

```diff
@@ -98,12 +99,23 @@ import org.json4s.DefaultFormats
 implicit val formats: Formats = DefaultFormats ++ JodaTimeSerializers.all
 ```
 
+### PlayJson
```

Comment:
I think we should also mention that `finch-playjson` requires `Reads[Foo]` ,`Writes[Foo]` to be defined for a type we want to serialize/deserialize. Let's mention that.